src/blue.ts

```diff
@@ -152,7 +168,7 @@ export function blueProxyFactory(env: MembraneBroker) {
         apply(shadowTarget: BlueShadowTarget, blueThisArg: BlueValue, blueArgArray: BlueValue[]): BlueValue {
             const { target } = this;
             const redThisArg = env.getRedValue(blueThisArg);
-            const redArgArray = env.getRedValue(blueArgArray);
+            const redArgArray = getStaticRedArray(blueArgArray);
```

Comment:
Should `getStaticRedArray` be on the `env` object?

Reply:
we only expose one method to env from each side, getRedValue and getBlueValue, the rest are internal to each side of the membrane.... getRedValue will under the hood, call the local getStaticRedArray if needed.